File: _persist/temp/venda/CSharp/Ncm.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("NCM")]
    public class Ncm : CollectionItem
    {
        [Campo("CD_NCM", CampoTipo.tfKey)]
        public string Cd_Ncm { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_NCM", CampoTipo.tfReq)]
        public string Ds_Ncm { get; set; }
    }
}

File: _persist/temp/venda/Java/Ncm.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("NCM")
public class Ncm extends CollectionItem
{
    @Campo("CD_NCM", CampoTipo.tfKey)
    public String Cd_Ncm;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("DS_NCM", CampoTipo.tfReq)
    public String Ds_Ncm;
}